packages/veraswap-sdk/src/swap/getTransaction.ts

```diff
@@ -346,6 +365,28 @@ export async function getTransaction(
                 });
             }
 
+            if (bridgeCurrencyIn.symbol && bridgeCurrencyIn.symbol in STARGATE_TOKEN_POOLS) {
+                if (!stargateQuote || stargateQuote.type !== "TOKEN") {
+                    throw new Error(`Stargate token quote is required for ${bridgeCurrencyIn.symbol} bridging`);
+                }
+
+                const pools = STARGATE_TOKEN_POOLS[bridgeCurrencyIn.symbol as keyof typeof STARGATE_TOKEN_POOLS];
+                const poolAddress = pools[bridgeCurrencyIn.chainId as keyof typeof pools];
```

Comment:
Also, I think we can assume that poolAddress is defined if we go a quote. so imo, replace with `pools[bridgeCurrencyIn.chainId as keyof typeof pools]!`